mcf-browser/src/App.tsx

```diff
@@ -127,7 +136,8 @@ class App extends Component {
    * param.
    */
   parseUrl() {
-    const params = new URLSearchParams(window.location.hash.trim('#'));
+    console.log('hash: ' + window.location.hash);
```

Comment:
please remove console.log

Reply:
Removed all instances of console.log throughout the project in the next commit!